### DISCUSSION TITLE
Accessing account info from the app

### DISCUSSION THREAD

**Discussion by Ella-eln** [community/community#132669] [Mobile]
How can i find details about my account from the app? Is that even possible?

**Comment by davevad93** [comment]
Hi @Ella-eln ,

If by “details about my account” you mean accessing account settings or editing profile info, that's not possible with the mobile app. You can manage pull requests, issues, interact with users, repositories, discussions, organizations, but you can't access or modify account details such as public profile, emails or passwords.

**Reply by Ella-eln** [reply]
Yes, that is what I meant

**Reply by davevad93** [reply]
@Ella-eln 👍 , The GitHub mobile app is just for managing a few things and to have a quick alternative to manage your repos for example, when you are away from home and don't have access to your computer; however it doesn't allow full control over your account like the web version.
